logstash-core/lib/logstash/instrument/collector.rb

```diff
@@ -29,7 +29,7 @@ def push(*args)
     end
 
     def self.snapshot_rotation_time=(time)
-      @snapsho_rotation_time = ime
+      @snapshot_rotation_time = time
```

Comment:
I recommend we add even simple ones for now, this would catch stupid stuff like this.